src/shared-components/EventForm/RecurrenceDropdown/RecurrenceDropdown.spec.tsx

```
@@ -30,88 +36,236 @@ describe('RecurrenceDropdown', () => {
+      );
+    });
+  });
+});
+
+describe('RecurrenceDropdown Edge Cases', () => {
+  beforeEach(() => {
+    vi.resetModules();
+    vi.doMock('shared-components/DropDownButton', () => ({
+      __esModule: true,
+      default: ({ onSelect }: { onSelect: (value: string) => void }) => (
+        <button data-testid="trigger-invalid" onClick={() => onSelect('999')}>
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)